doc/smart-switch/high-availability/smart-switch-ha-dpu-scope-dpu-driven-setup.md

```diff
@@ -265,17 +307,17 @@ sequenceDiagram
 
 ### 8.3. Planned switchover
 
-In DPU-driven setup, switchover is done via shutdown one side of the DPU, and DPUs pair need to be able to handle the switchover internally.
 
 ### 8.4. ENI migration / HA re-pair
 
 To support things like upgrade, we need to update the HA set to pair with another DPU. In this case, the following steps needs to be performed step by step:
 
 1. Trigger [Planned shutdown](#82-planned-shutdown) on the DPU that needs to be removed from the HA set.
-2. Update HA set information on all switches.
+2. Update HA set information on all switches and the corresponding DPUs.
    - This will cause the tables and objects related to old HA set to be removed and new HA set to be created.
-   - Once new HA set is programmed, [HA set creation](#81-ha-set-creation) workflow will be triggered and form the new HA set.
+   - The new DPU joining the HA pair will be in Dead state at this point.
```

Comment:
need to explain how the new DPU is joining the HA pair.

i am seeing you have removed the link to HA set creation, so I suppose it doesn't work.

Reply:
added the old link back. guess i removed it by mistake